media-gfx/hydrus/hydrus-565.ebuild

```diff
@@ -0,0 +1,155 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="sqlite"
+
+DOCS_BUILDER=mkdocs
+DOCS_DEPEND="dev-python/mkdocs-material dev-python/regex"
+
+inherit python-single-r1 desktop docs optfeature
+
+DESCRIPTION="A booru-like media organizer for the desktop"
+HOMEPAGE="https://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/hydrusnetwork/hydrus.git"
+else
+	SRC_URI="
+		https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		https://github.com/vaartis/hydrus/commit/5d2505a1eb4721f7750861f7622c46adec31db8b.diff -> hydrus-3500-mpeg-error.patch
+		https://github.com/vaartis/hydrus/commit/dc20196f47ac29568a40cd04a45cf4fc5f237603.diff -> hydrus-test-fixes.patch
```

Comment:
I would prefer versioned distfiles, but hopefully these are merged to next release.

Reply:
Unfortunately I'm not sure if these can be merged upstream, I have no idea what the first test is even supposed to do honestly, and the second one is for avoiding tests that require a pypi dependency that is just stupidly complicated to package.